core/primitives/src/views.rs

```diff
@@ -302,6 +302,8 @@ pub struct QueryResponse {
 #[cfg_attr(feature = "deepsize_feature", derive(deepsize::DeepSizeOf))]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct StatusSyncInfo {
+    pub epoch_id: EpochId,
+    pub epoch_start_height: BlockHeight,
```

Comment:
Do we need to add `serde::default()` or `Option` here to maintain compatibility?

Reply:
When deserializing a `StatusSyncInfo` without these fields, I think the default values of `EpochId` and `BlockHeight` could be misleading. So I went with `Option`.